### PR TITLE
implement Apple Home Page Clone

### PR DESCRIPTION
**Pull request by pointerish**
We have implemented the following:

- Background with image
- Navbar with gradient
- Footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    <link rel="shortcut icon"
+        href="https://freepngimg.com/download/apple/58686-logo-computer-apple-icons-free-transparent-image-hq.png"
+        type="image/x-icon">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="main-wrapper">
+        <div class="nav-text-section">
+            <nav>
```
Comment (javitocor) [inline index.html]: you can use a `header` tag here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                <img src="assets/footer-image2.jpg" alt="">
+            </div>
+            <div class="image-bottom">
+                <img src="assets/footer-image3.jpg" alt="">
+            </div>
+            <div class="image-bottom">
+                <img src="assets/footer-image4.jpg" alt="">
+            </div>
+        </footer>
+        <div class="bottom-footer">
```
Comment (javitocor) [inline index.html]: here also can use a `footer` tag

File: style.css [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  grid-template-rows: 36px;
+  border-radius: 4px;
+  justify-items: stretch;
+  margin-top: 90px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+nav div {
+  position: relative;
```
Comment (javitocor) [inline style.css]: it is a better practice to use classes instead of main selectors

File: style.css [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+.text-images img {
+  width: 600px;
+  margin-left: 33%;
+  margin-top: 5%;
+  margin-bottom: -4%;
+}
+
+footer {
+  display: inline-flex;
```
Comment (javitocor) [inline style.css]: same here

File: style.css [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+  border-radius: 10px;
+  background: linear-gradient(to bottom, rgba(122, 122, 122, 1) 0%, rgba(122, 122, 122, 1) 50%, rgba(77, 77, 77, 1) 51%, rgba(80, 80, 80, 1) 60%, rgba(77, 77, 77, 0.8) 100%);
+}
+
+input:focus {
+  outline: none;
+}
+
+nav a span {
+  text-align: center;
```
Comment (javitocor) [inline style.css]: same here

File: style.css [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+}
+
+.search-icon {
+  position: absolute;
+  margin-top: 12px;
+  margin-left: 5px;
+}
+
+nav div input {
+  width: 80%;
```
Comment (javitocor) [inline style.css]: same here

File: index.html [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+    <script src="https://unpkg.com/ionicons@5.0.0/dist/ionicons.js"></script>
+    <link rel="shortcut icon"
+        href="https://freepngimg.com/download/apple/58686-logo-computer-apple-icons-free-transparent-image-hq.png"
+        type="image/x-icon">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="main-wrapper">
+        <div class="nav-text-section">
```
Comment (javitocor) [inline index.html]: you can use a `main` tag in here

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello team! Good job with its project so far! 👍 
The navbar is almost a copy of the original, well done! ⭐ 
You have clean code and a great readme file 👏 
Good job setting up the linters 🟢 

However, you need to make some changes before approving the project! 💪 
Kindly, apply the required changes and resubmit your project for a new code review ⏩ 

Check my comments below 👇 

Good luck, guys! 😃 

- There is a problem on your website, this is how it looks the live link. I have created a new one and the result is the same, also with different screen sizes
Capture1
Capture

- Kindly fix the size of the main section and the background image
      -The CSS background should `cover` the whole page irrespective of the screen size.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/background_size_requirement.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/))

- Remove the horizontal scrollbar
- Add the text in the main section
- Add a hover effect to the navbar links

[OPTIONAL]
- improve your commit messages and commit more often, it is a good practice and will help you to follow up all your work

**Comment by pointerish** [comment]
Hi TSEs,

I submitted a new review request for the changes requested. I used the Chrome tool to emulate other resolutions since I'm on 1080p and the previous sizings looked okay for me.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+                <div class="nav-gradient nav-button">
+                    <ion-icon class="search-icon" name="search"></ion-icon>
+                    <input class="search-box" type="text">
+                </div>
+            </nav>
+            <div class="text-images">
+                <img src="assets/first-text.jpg" alt="First Text Image">
+                <img src="assets/second-text.png" alt="Second Text Image">
+                <img src="assets/third-text.png" alt="Third Text Image">
+            </div>
```
Comment (acushlakoncept) [inline index.html]: Please this is strongly discouraged as the purpose of this project is to master how to position elements using CSS.
 ### Use text and not a screenshot of the text, header element preferably
Screen Shot 2020-09-28 at 9 47 55 PM
Screen Shot 2020-09-28 at 9 48 03 PM
Screen Shot 2020-09-28 at 9 48 09 PM

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

#### Hi, @pointerish and @che30,

Great job so far 👍, there are still changes that are necessary for this project to be approved.

Please see the [original page](https://web.archive.org/web/20140301004610/http://www.apple.com/) for proper reference

- [ ] The CSS background should cover the whole page irrespective of the screen size using the background-size and position properties.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/background_size_requirement.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/)). See [original archive page](https://web.archive.org/web/20140301004610/http://www.apple.com/). You can use `background-size: cover`  
app


I have made some comments below on your code, please work on them.

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello!! Great effort!!

:100: I like you used media queries even if it wasn't required.

## Required Changes
### [Mandatory] :red_circle: 

- [x] ```<header>``` and ```<footer>``` should be outside the ```<main>``` container. These sections should follow this structure.
```
...
  <header>
    <nav>
      ... nav elements ...
    </nav>
  </header>
  
  <main>
    ... content ...
  </main>

  <footer>
    ... footer elements ...
  </footer>
```

- [x] Navbar should be an unordered list,  with the ```<a>``` tag wrapped around ```<li>``` tag. See inline comments

- [x] The bottom menu should also be an unordered list,  with the ```<a>``` tag wrapped around ```<li>``` tag. See inline comments

- [x] There should be only one ```<footer>```. Merge both footers into one. I know this was a previous request. The issue comment wasn't clear enough but was meant to wrap that section in the footer, not to create a new footer section. I apologize for the misunderstanding.

### [OPTIONAL]  :large_blue_circle: 

- [x] Prefer custom CSS selectors over general ones. They are more specific and easier to identify. It helps readability. Example:
**Actual**:
```
.navbar > a:hover {
  ... some properties...
}
```
**Improved** :+1: :
```
.navbar__link:hover {
  ... some properties...
}
```

- [x] Keep your code Dry. Example:
**Actual**:
```
.right-footer-container > a {
  text-decoration: none;
  color: #08c;
}

.left-footer > p > a {
  text-decoration: none;
  color: #08c;
}
```
**Improved** :+1: :
```
.right-footer-container > a,
.left-footer > p > a {
  text-decoration: none;
  color: #08c;
}
```

### [Comments] :green_circle: 

- The website it is not displaying correctly on smaller screens and is making it look distorted. This observation is out of this project scope and requirements, still is worth mentioning. It a great habit to use the browser inspector mode, and the media query inspector to avoid these issues and see if the page looks as intended in different screen sizes. 

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @pointerish and @che30
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly work on the suggested changes 👇.

### Header
- [ ] Please Fix the issue as prious TSE explained

|[Source Page](https://web.archive.org/web/20140301004610/http://www.apple.com/)|[Clone Page](https://che30.github.io/Apple-Clone-Microverse/)|
|:---:|:---:|
|source-page|clone|



##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by pointerish** [comment]
Hi @umairarshadbutt 

I made the changes as requested by the previous TSE. These are screenshots of our Live Demo link on 1080, 720 and 768 resolutions:
https://file.io/Ed5Jcyfl0xnK

Check the Live Demo using the device emulator on the dev tools and let me know how it looks since I guess we are having resolution discrepancies here.

**Comment by umairarshadbutt** [comment]
Hi @pointerish  
Thanks for reaching me. I reviewed it again. This link (https://file.io/Ed5Jcyfl0xnK) is not working so I'm attaching the screenshots from the live link.

|720px|768px|1080px|
|:---:|:---:|:---:|
||||

**Comment by pointerish** [comment]
Hi @umairarshadbutt 

Apologies for the broken link. Here's the zip on Google Drive: https://drive.google.com/file/d/15Zik302Mqvujvf16Ss3fQL_W8tGBrWpU/view?usp=sharing

Also, the 720, 768 and 1080 are meant to be the second value for the resolution, not the first one.

The values are 1366x720, 1366x786 and 1920x1080.

**Comment by umairarshadbutt** [comment]
Hi @pointerish
You can submit for re-review

**Comment by pointerish** [comment]
As discussed with @umairarshadbutt this site has been tested on 14xx width resolutions and it's behaving as requested. I will proceed to request a new code review.

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

## Congratulations team! I admire the effort you put on the project. 

You have corrected all previously required changes and now you may merge to master!

## See you at the top :rocket:!!

**Pull request merged**